.github/workflows/concrete_python_release_gpu.yml

```diff
@@ -1,61 +1,58 @@
-name: Concrete Python Release (GPU)
-
+name: concrete-python release-gpu
 on:
   workflow_dispatch:
-    inputs:
-      instance_id:
-        description: 'Instance ID'
-        type: string
-      instance_image_id:
-        description: 'Instance AMI ID'
-        type: string
-      instance_type:
-        description: 'Instance product type'
-        type: string
-      runner_name:
-        description: 'Action runner name'
-        type: string
-      request_id:
-        description: 'Slab request ID'
-        type: string
-      user_inputs:
-        description: 'either "nightly" or "public" or "private" to specify the release type'
-        required: true
-        default: 'nightly'
-        type: string
+  push:
+    tags:
```

Comment:
You may want to have a mechanism to ensure the one which push a tag is member of Zama organization.

Reply:
Not a high priority but could be worth if more external contributions are made to the code.